_pages/about-us/teams/acqstack.md

```diff
@@ -8,22 +8,22 @@ Making procurement joyful
 
 Welcome to the 18F Acquisition Chapter — we’re happy you’re here! We’ve compiled some information to help you get started.  We’ll update this guide as we receive feedback, so don’t be shy to share your thoughts. 
 
-### Who we are
+### Who We Are
```

Comment:
I'm 99% sure we use sentence case and this shouldn't change. @jenniferthibault can you confirm?

Reply:
Yep! We generally [use sentence case for headings](https://content-guide.18f.gov/capitalization/#headings)

Reply:
Corey knows best!:tm: